### PR TITLE
Use pills to make call number browse more obvious

### LINKED ISSUE #544: Come up with better call number browse link label on show view
Users are skipping over the "[Browse]" link in the record show view, partly because it's not prominently displayed and partly because users don't know that clicking the link would allow the users to view items that would be shelved near the record. Some suggestions of replacement text:
"More like this"
"View items shelved nearby"
"Virtual shelf browse"

### PR DESCRIPTION
**Pull request by escowles**
Closes #544

### CODE REVIEW

**Comment by escowles** [comment]
I experimented with using pills for the Browse links in the search results too.  But the pills add a lot of vertical space, so they either overlap or add a lot of spacing.  They also compete with the availability badges.  So I left those as-is, and just updated the original problem call number browse.

**Comment by jpstroop** [comment]
Could you attach a screenshot?

**Comment by axamei** [comment]
screen shot 2016-06-01 at 11 57 34 am

This could end up like the map marker needing the "Where to find it" text and a line break. Do you mind if I take a stab at this?

**Comment by escowles** [comment]
@axamei: go ahead — looks like there's also a feature test I didn't update...

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -36,16 +36,16 @@ def search_links(electronic_access)
...
+    if DONT_FIND_IT.include?(library)
       ''
     else
-      ' ' + link_to(%(<span class="map-text">#{t('blacklight.holdings.stackmap')}</span> <span class="glyphicon glyphicon-map-marker"></span>).html_safe, "#{ENV['stackmap_base']}?loc=#{location}&id=#{bib}", :target => '_blank', class: 'find-it', 'data-map-location' => location.to_s, 'data-toggle' => 'tooltip')
+      ' ' + link_to(%(<span class="link-text">#{t('blacklight.holdings.stackmap')}</span> <span class="glyphicon glyphicon-map-marker"></span>).html_safe, "#{ENV['stackmap_base']}?loc=#{location}&id=#{bib}", :target => '_blank', class: 'find-it', 'data-map-location' => location.to_s, 'data-toggle' => 'tooltip')
     end
   end
 
-  def locate_link_with_gylph(location, bib, call_number, library = nil)
-    if DONT_FIND_IT.include?(library) || call_number.blank?
```
Comment (tampakis) [inline app/helpers/application_helper.rb]: or these

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -103,11 +103,11 @@ def process_physical_holding(holding, bib_id, holding_id, is_journal)
                               holding_id: holding_id
                             }
                             )
-      location << locate_link(holding['location_code'], bib_id, holding['call_number'], holding['library']).html_safe
```
Comment (tampakis) [inline app/helpers/application_helper.rb]: revert to this

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -203,7 +203,7 @@ def holding_block_search(document)
   def search_location_display(holding, document)
     render_arrow = (!holding['library'].blank? && !holding['call_number'].blank?)
     arrow = render_arrow ? ' &raquo; ' : ''
-    location_display = holding['location'] + arrow + content_tag(:span, %(#{holding['call_number']}#{locate_link_with_gylph(holding['location_code'], document['id'], holding['call_number'], holding['library'])}).html_safe, class: 'call-number')
```
Comment (tampakis) [inline app/helpers/application_helper.rb]: restore this line

File: app/helpers/application_helper.rb [hunk 4]
```diff
@@ -234,14 +234,14 @@ def subjectify(args)
         full_sub = sub_array[i][j]
       end
       lnk += '  '
-      lnk += link_to('[Browse]', "/browse/subjects?q=#{full_sub}", class: 'browse-subject', 'data-toggle' => 'tooltip', 'data-original-title' => "Browse: #{full_sub}", title: "Browse: #{full_sub}", dir: full_sub.dir.to_s)
```
Comment (tampakis) [inline app/helpers/application_helper.rb]: revert to this

File: app/helpers/application_helper.rb [hunk 5]
```diff
@@ -234,14 +234,14 @@ def subjectify(args)
...
+      lnk += link_to('[Browse]', "/browse/subjects?q=#{full_sub}", class: 'browse-subject', 'data-toggle' => 'tooltip', 'data-original-title' => "Browse: #{full_sub}", title: "Browse: #{full_sub}", dir: getdir(full_sub).to_s)
       args[:document][args[:field]][i] = lnk.html_safe
     end
   end
 
   def browse_name(args)
     args[:document][args[:field]].each_with_index do |name, i|
-      newname = link_to(name, "/?f[author_s][]=#{name}", class: 'search-name', 'data-toggle' => 'tooltip', 'data-original-title' => "Search: #{name}", title: "Search: #{name}") + '  ' + link_to('[Browse]', "/browse/names?q=#{name}", class: 'browse-name', 'data-toggle' => 'tooltip', 'data-original-title' => "Browse: #{name}", title: "Browse: #{name}", dir: name.dir.to_s)
```
Comment (tampakis) [inline app/helpers/application_helper.rb]: revert to this

File: app/helpers/application_helper.rb [hunk 6]
```diff
@@ -107,7 +107,7 @@ def process_physical_holding(holding, bib_id, holding_id, is_journal)
       info << content_tag(:h3, location.html_safe, class: 'library-location')
     end
     unless holding['call_number'].blank?
-      cn_browse_link = link_to('[Browse]', "/browse/call_numbers?q=#{holding['call_number_browse']}", class: 'browse-cn', 'data-toggle' => 'tooltip', 'data-original-title' => "Browse: #{holding['call_number_browse']}", title: "Browse: #{holding['call_number_browse']}")
+      cn_browse_link = link_to('<span class="link-text">Browse nearby items</span> <span class="icon-bookslibrary"></span>'.html_safe, "/browse/call_numbers?q=#{holding['call_number_browse']}", class: 'browse-cn', 'data-toggle' => 'tooltip', 'data-original-title' => "Browse: #{holding['call_number_browse']}", title: "Browse: #{holding['call_number_browse']}")
```
Comment (tampakis) [inline app/helpers/application_helper.rb]: I have a feeling that the label for the call number browse is something that's going to be changed a couple of times. Could we refer to it via the I18n locale config?

File: spec/helpers/holding_block_spec.rb [hunk 7]
```diff
@@ -134,7 +134,7 @@
         expect(show_result.last).to include library
       end
       it 'link to call number browse' do
-        expect(show_result.last).to have_link('[Browse]', href: "/browse/call_numbers?q=#{call_number}")
+        expect(show_result.last).to have_link('Browse nearby items', href: "/browse/call_numbers?q=#{call_number}")
```
Comment (tampakis) [inline spec/helpers/holding_block_spec.rb]: Update the spec to refer the locale value as well so changes to the label text don't breaking the spec.

**Comment by tampakis** [comment]
Looks good!

**Pull request merged**